modules/administration_manual/pages/configuration/files/encryption/index.adoc

```diff
@@ -0,0 +1,132 @@
+
+[cols="20%,80%",options="header",]
+|===
+| Directory
+| Description
+
+| `data/<user>/files_encryption`
+| Users' private keys and all other keys necessary to decrypt the users' files.
+
+| `data/files_encryption`
+| Private keys and all other keys necessary to decrypt the files stored on a system wide external storage.
+|===
+
+NOTE: You can move the keys to a different location. To do so, refer to the xref:configuration/files/encryption/moving-key-locations.adoc[Move Key Location] section of the documentation.
+
+== Encryption Keys and Encrypted Data
+
+When encryption is enabled, all files are encrypted and decrypted by the ownCloud application, and stored encrypted on your remote storage. 
+This protects your data on externally hosted storage. 
+The ownCloud admin and the storage admin will see only encrypted files when browsing backend storage.
+
+Encryption keys are stored only on the ownCloud server, eliminating exposure of your data to third-party storage providers. 
+The encryption application does *not* protect your data if your ownCloud server is compromised, and it does not prevent ownCloud administrators from reading users' files.
+
+This would require client-side encryption, which this application does not provide. 
+If your ownCloud server is not connected to any external storage services, it is better to use other encryption tools, such as file-level or whole-disk encryption.
+
+[CAUTION]
+====
+SSL terminates at or before the webserver on the ownCloud server.
+Consequently, all files are in an unencrypted state between the SSL connection
+termination and the ownCloud code that encrypts and decrypts them.
+This is, potentially, exploitable by anyone with administrator access to your server.
+For more information, read: https://owncloud.org/blog/how-owncloud-uses-encryption-to-protect-your-data/[How ownCloud uses encryption to protect your data].
+====
+
+include::encryption-types.adoc[]
+
+[[before-enabling-encryption]]
+== Before Enabling Encryption
+
+Plan very carefully before enabling encryption, because it is not
+reversible via the ownCloud Web interface. If you lose your encryption
+keys, your files are *not* recoverable. Always have backups of your
+encryption keys stored in a safe location, and consider enabling all
+recovery options.
+
+You have more options via xref:configuration/server/occ_command.adoc#encryption[the `occ` command's encryption options].
+
+[IMPORTANT]
+====
+You can't manage encryption without access to the command line.
+If your ownCloud installation is on a hosted environment and you don't have access to the command line, you won't be able to run xref:configuration/server/occ_command.adoc[occ commands].
+In this case, *don't enable encryption*!
+====
+
+include::enable-encryption.adoc[leveloffset=+1]
+
+[[files-not-encrypted]]
+== Which Files Are Encrypted
+
+Only the data in the files in `data/<user>/files` are encrypted, and not the filenames or folder structures. 
```

Comment:
``Only the data in the files in `data/<user>/files` are encrypted, and not `` -->
``Only the data of the files in `data/<user>/files` and external storages if enabled are encrypted, not ``